src/Tracer.php

```diff
@@ -104,25 +103,14 @@ private function mapColumns(array $columns): array
             ->toArray();
     }
 
-    public static function columnAttributes($column): string
+    public static function columnAttributes(array $column): string
     {
         $attributes = [];
 
         $type = self::translations($column['type_name']);
 
-        if (in_array($type, ['decimal', 'float']) && str_contains($column['type'], '(')) {
-            $options = Str::between($column['type'], '(', ')');
-            if ($options) {
-                $type .= ':' . $options;
-            }
-        } elseif ($type === 'string' && str_contains($column['type'], '(')) {
-            $length = Str::between($column['type'], '(', ')');
-            if ($length != 255) {
-                $type .= ':' . $length;
-            }
-        } elseif ($type === 'enum') {
-            $options = Str::between($column['type'], '(', ')');
-            $type .= ':' . $options;
+        if (str_contains($column['type'], '(')) {
+            $type .= ':' . Str::between($column['type'], '(', ')');
```

Comment:
This will get comma-separated column's type attributes for `string`, `char`, `decimal`, `float`, `double`, `enum`, `set`, `time`, `timetz`, `datetime`, `datetimetz`, `timestamp`, `timestamptz`, `interval` and `bit`.